styles/tokens/autocomplete.json

```diff
@@ -0,0 +1,152 @@
+  {
+    "autocomplete": {
+      "results": {
+        "active": {
+          "background-color": {
+            "comment": "Background color of the active Autocomplete.",
+            "value": "{black-tint.10.value}",
+            "attributes": {
+              "category": "color"
+            },
+            "file": "settings",
+            "themable": true
+          }
+        },
+        "background-color": {
+          "comment": "Background color of the Autocomplete results.",
+          "value": "{white.value}",
+          "attributes": {
+            "category": "color"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "border-width": {
+          "comment": "The border size of the Autocomplete results.",
+          "value": "1px",
+          "attributes": {
+            "category": "size"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "border-style": {
+          "comment": "The border style of the Autocomplete results.",
+          "value": "solid",
+          "attributes": {
+            "category": "content"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "border-color": {
+          "comment": "The border color of the Autocomplete results.",
+          "value": "{gray.value}",
+          "attributes": {
+            "category": "color"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "border": {
+          "comment": "The border of the Autocomplete results.",
+          "value": "{autocomplete.results.border-width.value} {autocomplete.results.border-style.value} {autocomplete.results.border-color.value}",
+          "attributes": {
+            "category": "content"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "max-width": {
+          "comment": "The max-width of the Autocomplete results.",
+          "value": "27.5rem",
+          "attributes": {
+            "category": "size"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "max-height": {
+          "comment": "The max-height of the Autocomplete results.",
+          "value": "18rem",
+          "attributes": {
+            "category": "size"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "overflow-y": {
+          "comment": "The overflow-y behavior of the Autocomplete results.",
+          "value": "scroll",
+          "attributes": {
+            "category": "content"
+          },
+          "file": "settings",
+          "themable": true
+        },
+        "shadow": {
+          "comment": "The box shadow of the Autocomplete results.",
+          "value": "0 0 40px 2px rgba(0, 0, 0, 0.1)",
```

Comment:
Do we have this box-shadow value set as a variable anywhere?

Reply:
I copied/renamed this from the Dropdown component. We have shadows defined for a bunch of components and they're all slightly different 😑